e2e/__snapshots__/routes.spec.ts.snap

```diff
@@ -66,13 +82,17 @@ exports[`generates the routes 1`] = `
       {
         path: \\":doc?\\",
```

Comment:
I think we should use single quotes to simplify the snapshots reading 😆 . I will change that later on